tide/master/MasterApplication.cpp

```diff
@@ -423,10 +440,16 @@ void MasterApplication::_initTouchListener()
                 if (_planarController)
                 {
                     if (_planarController->getState() == ScreenState::OFF)
+                    {
                         if (!_planarController->powerOn())
                             put_flog(LOG_INFO,
                                      "Could not power on the screens by "
                                      "touching the wall");
+                        else
```

Comment:
please reverse if-else so that the check if not a negative '!'

Also, remove the two levels of indentation in this lambda with an early out:
if (!_planarController || planarController->getState() == ScreenState::OFF)
     return;